test/cmd/diagnostics.sh

```diff
@@ -16,15 +16,15 @@ os::log::install_errexit
 # Without things feeding into systemd, AnalyzeLogs and UnitStatus are irrelevant.
 # The rest should be included in some fashion.
 
-os::cmd::expect_success 'openshift ex diagnostics ClusterRoleBindings ClusterRoles ConfigContexts '
+os::cmd::expect_success 'oadm diagnostics ClusterRoleBindings ClusterRoles ConfigContexts '
```

Comment:
Add a specific check to make sure that we don't remove `openshift ex diagnostics` by accident and add a comment to keep people from removing it until we're ready.